Wikipedia/Code/SectionEditorViewController.swift

```diff
@@ -46,6 +48,18 @@ class SectionEditorViewController: UIViewController {
         apply(theme: theme)
 
         WMFAuthenticationManager.sharedInstance.loginWithSavedCredentials { (_) in }
+    
+        NotificationCenter.default.addObserver(forName: UIWindow.keyboardDidChangeFrameNotification, object: nil, queue: nil, using: { [weak self] notification in
```

Comment:
This observation is never removed, so even though `self` is weak, it continues to fire with self as nil even after the VC is deallocated.

Reply:
Ah thx. We may have same issue in `ViewController.swift`. I can fix it there when I fix this.